core/ibft_test.go

```diff
@@ -2128,8 +2136,14 @@ func TestIBFT_WatchForFutureRCC(t *testing.T) {
 	ctx, cancelFn := context.WithCancel(context.Background())
 	defer cancelFn()
 
+	wg.Add(1)
+
 	go func() {
 		defer cancelFn()
+		defer func() {
+			cancelFn()
```

Comment:
Could we remove previous line (2242) of the code?

Reply:
Good catch 💯 

It's leftover 🤦‍♂️ 

Removed:
[2476b39](https://github.com/0xPolygon/go-ibft/pull/29/commits/2476b390ffbc5cd6be7f9ca3a20070fa6d8aacf6)

Reply:
Thank you!